services/orchestrator-service/src/component.ts

```diff
@@ -40,6 +41,7 @@ export class OrchestratorServiceComponent implements Component {
       enableHttpAccessLog: true,
     });
     application.component(LoggingComponent);
```

Comment:
If we use our core component then this logger should not be needed. Our core logger should be used.

Reply:
@Tyagi-Sunny coreComponent already has 'LoggerExtensionComponent'